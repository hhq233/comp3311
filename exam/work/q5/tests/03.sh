python3 q5.py "Gold Coast"
