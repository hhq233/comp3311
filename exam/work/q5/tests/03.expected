Balter Brewing Co, who brewed 4 beers in 2019, 2020 and 2021
Black Hops, who brewed 4 beers in 2020
